browser/src/control/Control.UIManager.js

```diff
@@ -368,7 +370,6 @@ L.Control.UIManager = L.Control.extend({
 
 		if (this.map.isPresentationOrDrawing()) {
 			// remove unused elements
-			L.DomUtil.remove(L.DomUtil.get('spreadsheet-toolbar'));
```

Comment:
I think these are no harm? This one and the one in `docType === 'text'`.

Reply:
Unless we have a good reason, i think it maybe better to remove them when loading different files other than Calc.

Reply:
I thought that if we hide it we don't need to remove it, similar as we are doing for `formulabar-row` but yeah probably leaving the remove is a more conservative way to go about it